src/test/kotlin/MempoolTest.kt

```diff
@@ -0,0 +1,52 @@
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.serialization.kotlinx.json.*
+import kotlinx.coroutines.runBlocking
+import kotlinx.serialization.json.Json
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class FetchBlockIdTest {
+
+    @Test
+    fun testFetchFirstBlockId() = runBlocking{
+        val client = HttpClient(CIO) {
+            install(ContentNegotiation) {
+                json(Json {
+                    ignoreUnknownKeys = true
+                    prettyPrint = true
+                    isLenient = true
+                })
+            }
+        }
+
+        try {
+            val firstBlockId = client.get("https://mempool.space/api/v1/blocks/730000").body<List<Block>>().first()
+            assertEquals("0000000000000000000384f28cb3b9cf4377a39cfd6c29ae9466951de38c0529", firstBlockId.id)
+        } catch (e: Exception) {
+            println("Error fetching first block ID: $e")
+        }
+    }
```

Comment:
Same here, these tests don't make any sense if you duplicate the code inside it! The tests should look like this:

```kotlin
@Test
    fun testFetchFirstBlockId() = runBlocking {
        val client = MempoolClient()
        val result = client.fetchFirstBlockId(730000)
        assertTrue(result.isSuccess)
        assertEquals("0000000000000000000384f28cb3b9cf4377a39cfd6c29ae9466951de38c0529", result.getOrThrow())
    }
```

You should also add tests for the failure cases (for example, providing an invalid blockId by calling `client.fetchFirstBlockId(10_000_000)`